src/prop.c

```diff
@@ -46,6 +46,7 @@ static const struct strtab typetab[] = {
   { "s64",     PT_S64_ATOMIC },
   { "dbl",     PT_DBL },
   { "time",    PT_TIME },
+  { "int",     PT_DYN_INT },
```

Comment:
What's this for?

Reply:
PT_"DYN_INT" is from Dynamic Integer --> is required in vaapi.c to load 'ui' and 'uilp'. It has to paired with .def.dyn_i.
![dyn_int](https://github.com/tvheadend/tvheadend/assets/4031821/f99d3629-c969-460d-8c2b-1ab25435b0dd)

The value has to be loaded from a function (that provide results from vinfo probing). 
Previously inode had only 'dynamic list' (that ca be loaded from a function).
![list](https://github.com/tvheadend/tvheadend/assets/4031821/a35acfce-97ed-46fb-aba2-587d0a5678f5)

I see that is not compiling if you don't have VAAPI ; we need to exclude vainfo.c / vainfo.h if VAAPI is not enabled.